feature-styles/US_EnvironmentalManagementFacility.se

```diff
@@ -1,52 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<se:FeatureTypeStyle xmlns:us-emf="http://inspire.ec.europa.eu/schemas/us-emf/4.0" 
+<se:FeatureTypeStyle xmlns:us-emf="http://inspire.ec.europa.eu/schemas/us-emf/4.0"
 xmlns:act-core="http://inspire.ec.europa.eu/schemas/act-core/4.0"
-xmlns:ogc="http://www.opengis.net/ogc" xmlns:se="http://www.opengis.net/se" 
-xmlns:sld="http://www.opengis.net/sld" xmlns:xlink="http://www.w3.org/1999/xlink" 
+xmlns:ogc="http://www.opengis.net/ogc" xmlns:se="http://www.opengis.net/se"
+xmlns:sld="http://www.opengis.net/sld" xmlns:xlink="http://www.w3.org/1999/xlink"
 xsi:schemaLocation="http://www.opengis.net/se http://schemas.opengis.net/se/1.1.0/FeatureStyle.xsd"
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.1.0">
-	<se:Name>US.EnvironmentalManagementSite.Default</se:Name>
+	<se:Name>US.EnvironmentalManagementFacility</se:Name>
 	<se:Description>
-    	<se:Title>Environmental Management Facilities</se:Title>
+    	<se:Title>Environmental Management Facility</se:Title>
```

Comment:
Is this missing the suffix "Installation Style" (see 11.2.2 in DS)

Reply:
HC doesn't support multiple styles for the same feature type, so we can't offer the installation style and the default style.

Reply:
Ok